gnome-shell/src/install-shell.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# -*- coding: UTF-8 -*-
+
+project_name="$1"
+prefix="${MESON_INSTALL_DESTDIR_PREFIX}/share"
+
+ln -sf "${prefix}/gnome-shell/theme/${project_name}" "${prefix}/themes/${project_name}/gnome-shell"
```

Comment:
I suggest to revert the direction of this symlink as explained in my comment for light/meson.build below.